test/form/samples/builtin-prototypes/object-expression/_expected.js

```diff
@@ -1,11 +1,11 @@
 const object = {};
 object.propertyIsEnumerable( 'toString' );
-{}.propertyIsEnumerable( 'toString' );
-{}.propertyIsEnumerable( 'toString' ).valueOf();
+({}).propertyIsEnumerable( 'toString' );
+({}).propertyIsEnumerable( 'toString' ).valueOf();
```

Comment:
@lukastaegert If all test output is reparsed with acorn in the test framework then issues like this one could be caught earlier in the future. Please ignore if this PR already implemented this suggestion, but I did not see it.

For example, using the previous version of this file:
```
$ rollup test/form/samples/builtin-prototypes/object-expression/_expected.js --silent
[!] Error: Unexpected token
test/form/samples/builtin-prototypes/object-expression/_expected.js (3:2)
1: const object = {};
2: object.propertyIsEnumerable( 'toString' );
3: {}.propertyIsEnumerable( 'toString' );
     ^
4: {}.propertyIsEnumerable( 'toString' ).valueOf();
Error: Unexpected token
```

Reply:
With 574050be1f3a7197945b8adb2624a10f496af693:

```
$ rollup --silent < test/form/samples/system-export-compact/_expected.js
[!] Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)
- (3:8)
1: System.register([],function(exports){'use strict';return{execute:function(){exports({a:void 0,b:void 0,c:void 0});let a, b;
2: 
3: function(v){return exports({a:a}),v}([{ b: a = (exports('a',a-1),a--) }] = { b: function(v){return exports({b:v,c:v}),v}(--b) });}}});
           ^
```
```
$ rollup --silent < test/form/samples/system-multiple-export-bindings/_expected.js 
[!] Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)
- (47:12)
45: 
46:       // Update Expression
47:       function (v) { return exports({ a: v, a2: v }), v; }(++a), (exports('b', b + 1), b++), (++c, exports({ c: c, c2: c }), c);
                   ^
```